dist/profile/manifests/kubernetes/resources/chatbot_jenkinsadmin.pp

```diff
@@ -0,0 +1,63 @@
+#   Class: profile::kubernetes::resources::chatbot_jenkinsadmin
+#
+#   This class deploys the Jenkins Admin bot on IRC
+#
+#
+class profile::kubernetes::resources::chatbot_jenkinsadmin (
+    String $github_login,
+    String $github_password,
+    String $jira_login,
+    String $jira_password,
+    String $nick_password,
+    Array $clusters = $profile::kubernetes::params::clusters,
+    String $image_tag = 'latest'
+
+) inherits profile::kubernetes::params {
+
+  require profile::kubernetes::kubectl
+  require profile::kubernetes::resources::nginx
+  require profile::kubernetes::resources::lego
+
+  $clusters.each | $cluster | {
+    $context = $cluster['clustername']
+
+    file { "${profile::kubernetes::params::resources}/${context}/chatbot_jenkinsadmin":
+      ensure => 'directory',
+      owner  => $profile::kubernetes::params::user,
+    }
+
+    profile::kubernetes::apply { "chatbot_jenkinsadmin/namespace.yaml on ${context}":
+      context  => $context,
+      resource => 'chatbot_jenkinsadmin/namespace.yaml'
+    }
+
+    $github_credentials = "login=${github_login}\npassword=${github_password}\n"
+    $jira_credentials = "login=${jira_login}\npassword=${jira_password}\n"
+
+    profile::kubernetes::apply { "chatbot_jenkinsadmin/secret.yaml on ${context}":
+      context    => $context,
+      parameters => {
+        'github' => base64('encode', $github_credentials, 'strict'),
+        'jira'   => base64('encode', $jira_credentials , 'strict'),
+      },
+      resource   => 'chatbot_jenkinsadmin/secret.yaml'
+    }
+
+    profile::kubernetes::apply { "chatbot_jenkinsadmin/deployment.yaml on ${context}":
+      context    => $context,
+      parameters => {
+        'image_tag'     =>  $image_tag,
+        'nick_password' => $nick_password
```

Comment:
Is there a reason this isn't being set as a secret too?

Reply:
I'll update it